app/src/main/java/com/android/partagix/ui/screens/LoginScreen.kt

```diff
@@ -54,87 +69,197 @@ fun LoginScreen(authentication: Authentication, modifier: Modifier = Modifier) {
+                    R.drawable.ks28,
+                    R.drawable.remorque,
+                    R.drawable.brouette,
+                    R.drawable.gazon,
+                )
+            Box(modifier = Modifier.fillMaxWidth().graphicsLayer {}) {
+              ImageGrid(images = images, onShowSignInChange = { showSignIn = it })
+            }
+          }
+        }
+
+        Box(
+            modifier = Modifier.fillMaxWidth().padding(16.dp).align(Alignment.BottomCenter),
+            contentAlignment = Alignment.Center) {
+              Button(
+                  onClick = { showSignIn = true },
+                  modifier = Modifier.fillMaxWidth().height(60.dp).testTag("StartBorrowButton"),
+              ) {
+                Text(text = "Start Borrowing now", style = MaterialTheme.typography.titleLarge)
+              }
+            }
+      }
 
-    OutlinedButton(
-        modifier = modifier.testTag("PopUpLoginButton").fillMaxWidth().padding(16.dp),
-        onClick = { showSignIn = true },
-        shape = RoundedCornerShape(10.dp),
-        border = BorderStroke(1.dp, Color(0xffafafaf)),
-    ) {
-      Text(
-          text = "Start borrowing now",
-          style =
-              TextStyle(
-                  fontSize = 28.sp,
-                  fontFamily = FontFamily(Font(R.font.roboto)),
-                  fontWeight = FontWeight(500),
-                  color = Color(0xFF000000),
-                  textAlign = TextAlign.Center,
-              ),
-          modifier = modifier.padding(12.dp).testTag("LoginButtonOpenBottomSheet"))
-    }
-  }
   if (showSignIn) {
     ModalBottomSheet(
         onDismissRequest = { showSignIn = false },
         sheetState = sheetState,
-        modifier = modifier.fillMaxHeight(.35f),
+        modifier = modifier.fillMaxHeight(.4f).testTag("LoginBottomSheet"),
     ) {
-      Column(
+      LazyColumn(
           verticalArrangement = Arrangement.Top,
           horizontalAlignment = Alignment.CenterHorizontally,
-          modifier = modifier.fillMaxSize().testTag("LoginScreen2"),
-      ) {
-        OutlinedButton(
-            onClick = {
-              Log.w(TAG, "push button to sign in")
+          modifier = modifier.fillMaxSize().testTag("LoginScreen2")) {
+            item {
+              Text(
+                  text = "Sign in to start Borrowing.",
+                  style = TextStyle(fontSize = 24.sp, fontWeight = FontWeight.Bold),
+              )
+              Spacer(modifier = Modifier.height(20.dp))
+            }
+            item {
+              LoginOptionButton(
+                  modifier = Modifier.testTag("GoogleLoginButton"),
+                  name = "Sign in with Google",
+                  icon = R.drawable.google,
+                  onClick = {
+                    Log.w(TAG, "push button to sign in (Google)")
+                    authentication.signIn()
+                  })
+            }
+            item {
+              LoginOptionButton(
```

Comment:
The way you modularized code is really appreciated, but is it worth it to add another login button if it is useless ? 
I suggest you that you remove the second button from here, and the day we decide to add Facebook, Tequila or Email authentication we create a new PR that adds it here. What do you think ?

Reply:
Yeah I forgot to remove the second login option, it was to test the layout with a second login option for when we will add one